server/index.js

```diff
@@ -89,25 +89,112 @@ function getCurrentFixtures(round, currentSeason, leagueId) {
     return response.data.response;
   });
 }
+app.get('/api/team-form', (req, res, next) => {
+  const sql = `
+    select "matchDetails"
+    From "teamForm"
+    `;
+  db.query(sql)
+    .then(result => {
+      const dbresult = result.rows;
+      const flatten = dbresult.flat(1);
+      res.json(flatten);
+    })
+    .catch(err => next(err));
+});
+
 app.get('/api/week-games/:date', (req, res, next) => {
   const sql = `
    select "fixtures"
       from "weekGames"
 `;
   db.query(sql)
     .then(result => {
-      const dbresult = result.rows[0];
-      const todayGames = dbresult.fixtures;
-      const time = new Date();
-      const formatDate = dateFns.format(time, 'yyyy-MM-dd');
-      const filter = todayGames.filter(fixture => {
-        return fixture.fixture.date.slice(0, 10) === formatDate;
+      const dbresult = result.rows;
+      const todayGames = dbresult.map(fixtures => {
+        const filteredGames = fixtures.fixtures.filter(fixture => {
+          const time = new Date();
+          const formatDate = dateFns.format(time, 'yyyy-MM-dd');
+          return fixture.fixture.date.slice(0, 10) === formatDate;
+        });
+        return filteredGames;
       });
-      res.json(filter);
+      const flatten = todayGames.flat(1);
+      res.json(flatten);
+    })
+    .catch(err => next(err));
+});
+
+app.post('/api/team-form/:teamId', (req, res, next) => {
```

Comment:
This should be a `GET`, instead of a `POST` because the client wants to retrieve some data. You can add more than one parameter to your query string.

```js
app.get('/api/match-details', (req, res, next) => {
  const { homeId, awayId, leagueId, date } = req.query;
});
```

```js

fetch(`/api/match-details?homeId=${???}&awayId=${???}&leagueId=${???}&date=${???}`)
```